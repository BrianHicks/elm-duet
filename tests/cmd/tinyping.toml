bin.name = "elm-duet"
args = ["elm-duet.yaml"]
binary = true
