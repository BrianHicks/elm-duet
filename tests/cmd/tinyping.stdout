wrote elm.ts
wrote src/Main/Flags.elm
wrote src/Main/Ports.elm
